content/en/docs/instrumentation/js/manual.md

```diff
@@ -4,146 +4,349 @@ aliases:
+
+Before any other module in your application is loaded, you must initialize the
+SDK. If you fail to initialize the SDK or initialize it too late, no-op
+implementations will be provided to any library which acquires a tracer or meter
+from the API.
+
+<!-- prettier-ignore-start -->
+
+{{< tabpane langEqualsHeader=true >}}
+
+{{< tab TypeScript >}}
+/*instrumentation.ts*/
+import { NodeSDK } from '@opentelemetry/sdk-node';
+import { ConsoleSpanExporter } from '@opentelemetry/sdk-trace-node';
+import { PeriodicExportingMetricReader, ConsoleMetricExporter } from '@opentelemetry/sdk-metrics';
+const { Resource } = require('@opentelemetry/resources');
+const { SemanticResourceAttributes } = require('@opentelemetry/semantic-conventions');
+
+const sdk = new NodeSDK({
+  resource: new Resource({
+    [ SemanticResourceAttributes.SERVICE_NAME ]: "yourServiceName",
+    [ SemanticResourceAttributes.SERVICE_VERSION ]: "1.0",
+  }),
+  traceExporter: new ConsoleSpanExporter(),
+  metricReader: new PeriodicExportingMetricReader({
+    exporter: new ConsoleMetricExporter()
+  })
 });
 
-const resource =
-  Resource.default().merge(
-    new Resource({
-      [SemanticResourceAttributes.SERVICE_NAME]: "service-name-here",
-      [SemanticResourceAttributes.SERVICE_VERSION]: "0.1.0",
-    })
-  );
+sdk
+  .start()
+{{< /tab >}}
 
-const provider = new NodeTracerProvider({
-    resource: resource,
+{{< tab JavaScript >}}
+/*instrumentation.ts*/
+import { NodeSDK } from '@opentelemetry/sdk-node';
+import { ConsoleSpanExporter } from '@opentelemetry/sdk-trace-node';
+import { PeriodicExportingMetricReader, ConsoleMetricExporter } from '@opentelemetry/sdk-metrics';
+const { Resource } = require('@opentelemetry/resources');
+const { SemanticResourceAttributes } = require('@opentelemetry/semantic-conventions');
+
+const sdk = new NodeSDK({
+  resource: new Resource({
+    [ SemanticResourceAttributes.SERVICE_NAME ]: "dice-server",
+    [ SemanticResourceAttributes.SERVICE_VERSION ]: "0.1.0",
+  }),
+  traceExporter: new ConsoleSpanExporter(),
+  metricReader: new PeriodicExportingMetricReader({
+    exporter: new ConsoleMetricExporter()
+  })
 });
-const exporter = new ConsoleSpanExporter();
-const processor = new BatchSpanProcessor(exporter);
-provider.addSpanProcessor(processor);
 
-provider.register();
+sdk
+  .start()
 {{< /tab >}}
 
 {{< /tabpane>}}
 
 <!-- prettier-ignore-end -->
-<!-- markdownlint-restore -->
 
-Next, ensure that `tracing.js|ts` is required in your node invocation. This is
-also required if you're registering instrumentation libraries. For example:
+For debugging and local development purpose, the following example exports
+telemetry to the console. After you have finished setting up manual
+instrumentation, you need to configure an appropriate exporter to
+[export the app's telemetry data](/docs/instrumentation/js/exporters/) to one or
+more telemetry backends.
+
+The example also sets up SDK default attributes, which are necessary, and
```

Comment:
```suggestion
The example also sets up SDK default attributes, which are 
```

Reply:
Since service.name is required and service.version is optional but recommended saying that they are "beneficial" alone is not strong enough.

> The example also sets up SDK default attributes:

maybe we can live without both?

Reply:
Yeah hm... maybe "The example also sets up SDK default attributes, including the required service name" ?

Reply:
tried a different wording, please take another look and what you think.